### PR TITLE
implement new screen for web/rss feeds

### LINKED ISSUE #460: New Public channel info page
On the public channel info page there are no more icons on the top left and no more QR code.

Instead there is a logo and two buttons (Follow/Unfollow and Share)

https://zpl.io/b6gY4RK

### PR DESCRIPTION
**Pull request by vujevits**
Fixes #460 

Changes proposed in this pull request:
- implements new design for rss feed info page

### CODE REVIEW

File: src/containers/FeedInfoContainer.ts [hunk 1]
```diff
@@ -27,13 +28,15 @@ const mapStateToProps = (state: AppState, ownProps: { navigation: TypedNavigatio
     };
 };
 
-const updateNavParam = (feeds: Feed[], navigation: TypedNavigation) => {
+const getFeedToOpen = (feeds: Feed[], navigation: TypedNavigation) => {
     const feedUrl = navigation.getParam<'FeedInfo', 'feed'>('feed').feedUrl;
     const isFollowed = navigation.getParam<'FeedInfo', 'feed'>('feed').followed;
 
     const updatedFeed = feeds.find(feed => feed.feedUrl === feedUrl);
     if (updatedFeed != null && updatedFeed.followed !== isFollowed) {
```
Comment (agazso) [inline src/containers/FeedInfoContainer.ts]: Why don't you put the `updatedFeed.followed !== isFollowed` condition in the `find`? To me it's easier to understand that way.

File: src/App.tsx [hunk 2]
```diff
@@ -346,6 +346,9 @@ const Scenes: NavigationRouteConfigMap = {
     FeedInfo: {
         screen: FeedInfoContainer,
     },
+    WebFeedInfo: {
```
Comment (agazso) [inline src/App.tsx]: I would either call this `PublicFeedInfo` or maybe `RSSFeedInfo`

File: src/containers/FeedInfoContainer.ts (hunk 1 above)
Comment (vujevits) [reply src/containers/FeedInfoContainer.ts]: resolved in 272386a

File: src/App.tsx (hunk 2 above)
Comment (vujevits) [reply src/App.tsx]: resolved in 272386a

**Pull request merged**